cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java

```diff
@@ -144,6 +144,10 @@ public void removeNamespace(Id.Namespace namespace) throws ExploreException, SQL
     handleExploreFuture(futureSuccess, "remove", "namespace", namespace.getId());
   }
 
+  private boolean isDatasetExplorable(Id.DatasetInstance datasetInstance) {
```

Comment:
nit: The contract of this method seems to indicate that it operates on a dataset and returns if it is explorable. To check if explore is enabled in CDAP as a whole seems to be outside of the purview of this method. 

Only a nit though.